symengine/ruby/spec/basic_spec.rb

```diff
@@ -16,56 +16,5 @@
                 end
             end
         end
-
-        describe 'binary operations' do
-            before :each do
-                @a = SymEngine::Basic.new
-                @b = SymEngine::Basic.new
-            end
-            describe '#+' do
-                context 'with another Basic object as argument' do
-                    xit 'returns a Basic object that is result of
-                        self added to the argument' do
```

Comment:
These didn't test anything. The `xit` keyword marks them as pending. As you said, to test these methods, the `Basic` must be initialised. If we are initialising it with a `Symbol`, I thought it would be better to club it with other tests of `Symbol`.